languages/elixir/exercises/concept/booleans/.docs/after.md

```diff
@@ -0,0 +1,31 @@
+Elixir represents true and false values with the boolean type. There are only two values -- _true_ and _false_. These values can be bound to a variable and combined with boolean operators (`and/2`, `or/2`, `not/1`):
+
+```elixir
+is_true = true and true
```

Comment:
```suggestion
true_variable = true and true
```
Since `is_x` is usually reserved for boolean functions usable in guards, this may be more clear.

Reply:
sure